components/suggest/src/schema.rs

```diff
@@ -43,6 +57,12 @@ pub const SQL: &str = "
         ON DELETE CASCADE
     );
 
+    CREATE TABLE pocket_custom_details(
+        suggestion_id INTEGER PRIMARY KEY REFERENCES suggestions(id) ON DELETE CASCADE,
+        description TEXT NOT NULL,
```

Comment:
Looks like `description` is not included in the Pocket suggestion & the udl below, also looks like Firefox doesn't use it for now. Shall we exclude it from the schema here as well? We can always add it back later if needed.